src/Sarif.Viewer.VisualStudio/SdkUiUtilities.cs

```diff
@@ -802,6 +802,30 @@ public static bool TryGetWpfTextView(IVsTextView textView, out IWpfTextView wpfT
             return true;
         }
 
+        public static bool TryGetTextViewFromFrame(IVsWindowFrame frame, out ITextView textView)
+        {
+            IVsTextView vsTextView = VsShellUtilities.GetTextView(frame);
+
+            if (vsTextView == null)
+            {
+                textView = null;
+                return false;
+            }
+
+            object textViewHost;
+            Guid guidTextViewHost = Microsoft.VisualStudio.Editor.DefGuidList.guidIWpfTextViewHost;
+            if (ErrorHandler.Succeeded(((IVsUserData)vsTextView).GetData(ref guidTextViewHost, out textViewHost)) &&
+                textViewHost != null)
+            {
+                textView = ((IWpfTextViewHost)textViewHost).TextView;
+                return true;
+            }
+
+            textView = null;
+
+            return false;
+        }
+
         public static bool TryGetTextViewFromFrame(IVsWindowFrame frame, out IVsTextView vsTextView)
```

Comment:
>TryGetTextViewFromFrame [](start = 27, length = 23)

How much of this code is actually being used?

Reply:
Answer: A lot less than what is here.. Removed stuff we didn't need.

---
In reply to: [501943986](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r501943986) [](ancestors = 501943986)

Reply:
Yay! 700 lines gone.

---
In reply to: [502015022](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502015022) [](ancestors = 502015022,501943986)